src/scripts/main.js

```diff
@@ -11,18 +11,28 @@ const food = {
       'Cherry': {},
       'Strawberry': {},
     },
+
     'Yellow': {
       'Banana': {},
       'Pineapple': {},
     },
-
   },
 };
 
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    const li = document.createElement('li');
+
+    li.innerText = key;
+    ul.append(li);
+    element.append(ul);
```

Comment:
Move 'element.append(ul);' outside the for loop to avoid appending multiple times